hadoop-ozone/dist/src/main/compose/ozonesecure-mr/docker-compose.yaml

```diff
@@ -112,3 +112,13 @@ services:
       HADOOP_CLASSPATH: /opt/ozone/share/ozone/lib/hadoop-ozone-filesystem-lib-current-@project.version@.jar
       WAIT_FOR: rm:8088
     command: ["yarn","timelineserver"]
+  spark:
+    image: ahadoop/spark:v1
```

Comment:
Can you share the image details of ahadoop/spark:v1? Is it published in docker repo?

Reply:
Its based on our hadoop docker runner image. You can look at source at https://github.com/ajayydv/docker/tree/spark . We can create a team account for ozone in githib and docker and push this there.

Reply:
It's fine. Just use automated builds: https://docs.docker.com/docker-hub/builds/ 

For automated builds the source will be linked by dockerhub without any effort and easy to check the source.

(Or add a readme with a link to the github project, but automated builds can be better).

Reply:
@elek thanks for documentation link. Added source for automated builds in docker.